WS2012R2/lisa/setupscripts/install_kernel_rpm.ps1

```diff
@@ -25,6 +25,37 @@
 
 param([string] $vmName, [string] $hvServer, [string] $testParams)
 
+function Set-UbuntuBootKernel {
+    param (
+        [string] $ipv4,
+        [string] $sshkey
+    )
+
+    $retVal = $False
+    $script = @'
+#!/bin/bash
+
+cd /tmp/test-artifacts
+image_file=$(ls -1 *image* | grep -v "dbg" | sed -n 1p)
+if [[ "${image_file}" != '' ]]; then
+    kernel_identifier=$(dpkg-deb --info "${image_file}" | grep 'Package: ' | grep -o "image.*")
+    kernel_identifier=${kernel_identifier#image-}
+    sed -i.bak 's/GRUB_DEFAULT=.*/GRUB_DEFAULT="Advanced options for Ubuntu>Ubuntu, with Linux '$kernel_identifier'"/g' /etc/default/grub
+    update-grub
+else
+    exit 1
+fi
+'@
+
+    Set-Content -Value $script -Path ".\change-grub.sh"
```

Comment:
for powershell part, use a rootdir plus that path
use a variable for "change-grub.sh", and reuse it.